docs/data/data-grid/pages.ts

```diff
@@ -0,0 +1,55 @@
+const pages = [
+  {
+    pathname: '/x/react-data-grid',
+    title: 'Components',
+    icon: 'ToggleOnIcon',
+    children: [
+      {
+        pathname: '/x/react-data-grid',
+        subheader: '/x/react-data-grid/overview',
+        title: 'Overview',
+      },
+      { pathname: '/x/react-data-grid/demo' },
+      { pathname: '/x/react-data-grid/getting-started' },
+      { pathname: '/x/react-data-grid/migration-v4', title: 'Migration From v4' },
+      { pathname: '/x/react-data-grid/layout' },
+      { pathname: '/x/react-data-grid/columns' },
+      { pathname: '/x/react-data-grid/rows' },
+      { pathname: '/x/react-data-grid/editing' },
+      { pathname: '/x/react-data-grid/sorting' },
+      { pathname: '/x/react-data-grid/filtering' },
+      { pathname: '/x/react-data-grid/pagination' },
+      { pathname: '/x/react-data-grid/selection' },
+      { pathname: '/x/react-data-grid/events' },
+      { pathname: '/x/react-data-grid/export' },
+      { pathname: '/x/react-data-grid/components' },
+      { pathname: '/x/react-data-grid/style' },
+      { pathname: '/x/react-data-grid/localization' },
+      { pathname: '/x/react-data-grid/scrolling' },
+      { pathname: '/x/react-data-grid/virtualization' },
+      { pathname: '/x/react-data-grid/accessibility' },
+      { pathname: '/x/react-data-grid/group-pivot', title: 'Group & Pivot' },
+    ],
+  },
+  {
+    pathname: '/x/api/mui-data-grid',
+    title: 'API',
+    icon: 'CodeIcon',
+    children: [
+      { pathname: '/x/api/mui-data-grid', title: 'API Reference' },
```

Comment:
Same but `mui-`
Why not `react-data-grid` or `data-grid` ?

Reply:
Same reason as above.

Reply:
I'm resolving this conversation to keep a single thread